runner/cf-driver/run.sh

```diff
@@ -8,7 +8,8 @@ source "${currentDir}/base.sh"
 printf "[cf-driver] Using SSH to connect to %s and run '%s' step\n" "$CONTAINER_ID" "$2"
 
 # Add line below script shebang to source the /etc/profile
-sed -i '2isource /etc/profile \n' "$1"
+sed -i '2iPATH="$HOME/bin:$PATH"\n' "$1"
```

Comment:
[issue (potentially)] I think that /etc/environment source will need to be properly ordered, since I think it sets PATH without regard for any existing settings.

Reply:
That's what I figured you meant, but I didn't think through how git will decide what order they go in when it's merged until now--will have to find that out.